homeassistant/components/roborock/__init__.py

```diff
@@ -111,6 +111,9 @@ async def async_setup_entry(hass: HomeAssistant, entry: RoborockConfigEntry) ->
             translation_key="no_coordinators",
         )
     valid_coordinators = RoborockCoordinators(v1_coords, a01_coords)
+    await asyncio.gather(
```

Comment:
Can this be called from coordinator`_async_setup`? That method exists to allow setup that happens on the first call to the API

Reply:
We discussed over chat that this can't work because we need some stuff fetched in the first refresh.